app/javascript/modules/listingDetailsAside/ListingDetailsAside.tsx

```diff
@@ -34,28 +34,6 @@ export const ListingDetailsAside = ({ listing, imageSrc }: ListingDetailsSidebar
     </SidebarBlock>
   )
 
```

Comment:
**question (non-blocking):**  This is just a refactor right? Not related to How to Apply AC. Not a concern, just confirming!

Reply:
<img width="603" alt="Screenshot 2024-11-13 at 10 38 23 AM" src="https://github.com/user-attachments/assets/8f585d75-28e0-467d-932b-09c07b9c7867">
Style issues on rental and sales listing details. 

Happy to pair on this, the styles for the new GetHelpSidebar block are probably conflicting with something on the ListingDetailsAside

Reply:
Hi yes sorry I'm still working on this! Yeah I just want to be able to use the same component in both places so I don't have to recreate the wheel